apis/python/tests/test_reindexer_api.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+from tiledbsoma import build_index
+
+
+def main():
```

Comment:
Please rename this so that the function begins with `test` (ie. `test_indexer_basic`). This way it can be picked up by pytest automatically. If it appropriate to move this to `text_indexer.py` too then that would preferable as it would be nice to group similar tests together in the same file.